sdk/program/src/rent.rs

```diff
@@ -144,43 +166,21 @@ mod tests {
                 2,
                 1.2
             ),
-            (0, true)
+            RentDue::Exempt
         );
     }
 
-    #[ignore]
     #[test]
-    #[should_panic]
-    fn show_rent_model() {
```

Comment:
I also removed this test. It's been ignored since it was first committed, and I don't know what it could be testing. Did I miss something though?